lib/galaxy/workflow/modules.py

```diff
@@ -870,6 +870,12 @@ def callback( input, value, prefixed_name, prefixed_label ):
         jobs = execution_tracker.successful_jobs
         for job in jobs:
             self._handle_post_job_actions( step, job, invocation.replacement_dict )
+        if execution_tracker.execution_errors:
+            failed_count = len(execution_tracker.execution_errors)
+            success_count = len(execution_tracker.successful_jobs)
+            all_count = failed_count + success_count
+            message = "Failed to create %d out of %s job(s) for workflow step." % (failed_count, all_count)
+            raise Exception(message)
```

Comment:
Should we log the exception and raise it afterwards, like in the run.py or the other way around?